heron/instance/src/python/instance/BUILD

```diff
@@ -0,0 +1,17 @@
+package(default_visibility = ["//visibility:public"])
+
+load("/tools/rules/pex_rules", "pex_library", "pex_binary")
+
+pex_binary(
+  # build binary for single thread python heron instance
+  name = "heron-python-instance",
+  srcs = ["st_heron_instance.py"],
```

Comment:
@kramasamy I just thought if we are going to add double-thread heron instance later, it's better to distinguish between them.